npm-deps.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+function ncu() {
+    target=$1
+    npx npm-check-updates -u --target "${target}"
```

Comment:
We could prob add this to our `devDependencies` in the root `package.json` at some point since I imagine it might be a common thing we use (and thus avoid needing to use `npx`).

Reply:
ah yeah indeed that could be another way. Mmm no idea what's the best. I supposed npx is also quite common in a web environment right ?